pages/dev/governance/community-pool-proposals.mdx

```diff
@@ -12,35 +12,40 @@ We ask teams to create an application on our [community forum](https://community
 3. Once the feedback window has passed, you can submit the Proposal for an on-chain governance vote.
 
 ## On-chain governance proposal
-Once a Governance Forum proposal has been live for the required temperature check and the feedback window has closed, you can submit an on-chain proposal to be voted on. Governance proposals are submitted on-chain using `axelard` (the [Axelar CLI](/learn/cli)) and a local Axelar node.
+Once a Governance Forum proposal has been live for the required temperature check and the feedback window has closed, you can submit an on-chain proposal to be voted on. Governance proposals are submitted on-chain using `axelard` (the [Axelar CLI](/learn/cli)).
 
 ### Prerequisites
 
 * You must have [axelard](/node/config-node#download-binaries-and-initialize-configuration)
-* You must have a local Axelar node running
-* You must have an account named `validator` with at least **2000AXL** balance
+* You must have an account with at least **2,000AXL** balance
+* You must choose an [RPC endpoint](/resources), run an Axelar node
 
 
-### Create a validator account
+### Create a local wallet
+If you don't already have a local wallet, you can create one.
+
 ```bash
-# create an account validator
-$ axelard keys add validator 
+# create an account named `wallet`
+$ axelard keys add wallet 
 # enter passphrase
 
 # display key address 
 $ axelard keys list
 
-# send funds from an existing wallet to this validator account
-$ axelard tx bank send [account_with_fund_address] [validator_key_address] 2000000000uaxl
+# fund the wallet with 2,000 AXL
+
+# For example, if you have another local wallet, you could transfer the funds with:
+$ axelard tx bank send [account_with_fund_address] [wallet_address] 2000000000uaxl
```

Comment:
They can send funds via Keplr wallet, or get it from a CEX. They can check the balance on axelarscan. But they can use this command to send the funds back